src/utils/trends/data.js

```diff
@@ -38,14 +38,14 @@ export function findTimeOfDayBin(binSize, msPer24) {
 }
 
 /**
- * mungeDataForBin
+ * calculateStatsForBin
```

Comment:
https://en.wikipedia.org/wiki/Data_binning

It's a pretty standard term 😜 (So is the word "munge" among data scientists/statisticians, although I admit it is not very precise.)

Knowing that this method is being called per bin (as opposed to on the bins as a set) is important to me.